### PR TITLE
Add jetty collection to gz-collections.yaml

### PR DESCRIPTION
**Pull request by Crola1702**
As the title says

### CODE REVIEW

**Comment by j-rivero** [comment]
If I'm not wrong the appointed branches for Jetty are already setup in https://github.com/gazebo-tooling/gazebodistro/pull/181 and should be the same here. So no `main` branches but probably starting with Ionic branches.

**Comment by Crola1702** [comment]
> So no main branches but probably starting with Ionic branches.

Done. Can you check if everything is correct? @j-rivero

**Review by j-rivero — CHANGES_REQUESTED** [review]
I think that major_version are not in sync with the package names. I.e: gz-cmake4 branch for gz-cmake5 package. So we need to update the major_versions too.

File: jenkins-scripts/dsl/gz-collections.yaml [hunk 1]
```diff
@@ -441,67 +441,67 @@ collections:
       - name: gz-cmake
         major_version: 5
         repo:
-          current_branch: gz-cmake4
+          current_branch: gz-cmake5
```
Comment (azeey) [inline jenkins-scripts/dsl/gz-collections.yaml:444]: We should do the opposite here. Currently, Jetty should be the exact copy of Ionic.

**Review by j-rivero — APPROVED** [review]

**Pull request merged**